data-processing/kaapana-plugin/extension/docker/files/plugin/kaapana/operators/LocalDcm2JsonOperator.py

```diff
@@ -290,6 +290,7 @@ def replace_tags(self, dicom_meta):
                 vr = str(value['vr'])
 
                 if "nan" in value_str:
+                    # FIXME: doesn't this cause false positives, e.g. if "nanotech" appears in some string tag?
```

Comment:
I realized the probability for wrong matches is very low: `value_str` is still a *list* at this point, so 'nan' would have to appear as a single element in that list. (The conversion to a string is immediately below.)